server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -96,6 +96,7 @@ export function generateToken(
 	key: string,
 	scopes: ScopeType[],
 	user?: IUser,
+	jti?: string,
```

Comment:
For a lot of reasons, it would be ideal to add this param at the end of the list. To do that, simply
```ts
export function generateToken(
	tenantId: string,
	documentId: string,
	key: string,
	scopes: ScopeType[],
	user?: IUser,
    lifetime: number = 60 * 60,
	ver: string = "1.0",
	jti: string = uuid(),
): string {
  // ...
```

This also lets you delete the `const jwid ...` line, and instead use `{ jwtid: jti }` in the sign function